README.rst

```diff
@@ -1,38 +1,46 @@
-
-========
 Hdlparse
-========
+########
 
-Hdlparse is a simple package implementing a rudimentary parser for VHDL and Verilog. It is not capable of fully parsing the entire language. Rather, it is meant to extract enough key information from a source file to create generated documentation.
+Hdlparse is a simple package implementing a rudimentary parser for VHDL and Verilog.
+It is not capable of fully parsing the entire language.
+Rather, it is meant to extract enough key information from a source file to create generated documentation.
 
-This library is used by the `Symbolator <https://github.com/kevinpt/symbolator>`_ diagram generator.
+This library is used by the `Symbolator <https://github.com/hdl/symbolator>`_ diagram generator.
 
-For VHDL this library can extract component, subprogram, type, subtype, and constant declarations from a package. For Verilog it can extract module declarations (both 1995 and 2001 syntax).
+For VHDL this library can extract component, subprogram, type, subtype, and constant declarations from a package.
+For Verilog it can extract module declarations (both 1995 and 2001 syntax).
 
 
 Requirements
 ------------
 
 Hdlparse requires either Python 2.7 or Python 3.x and no additional libraries.
 
-The installation script depends on setuptools. The source is written in
-Python 2.7 syntax but will convert cleanly to Python 3 when the installer
-passes it through 2to3.
+The installation script depends on setuptools.
+The source is written in Python 2.7 syntax but will convert cleanly to Python 3 when the installer passes it through 2to3.
 
 
 Download
 --------
 
-You can access the Hdlparse Git repository from `Github
-<https://github.com/kevinpt/hdlparse>`_. You can install direct from PyPI with the "pip"
-command if you have it available.
+You can access the Hdlparse Git repository from `Github <https://github.com/hdl/pyHDLParser>`_.
```

Comment:
external URLs need 2x `_` at the end.

Reply:
In fact, I did not review the rst syntax. I did:

- Add `.btd.yml`, CI workflow and adapt `conf.py` for using the sphinx_btd_theme.
  - Remove sources for customising the previous theme (alabaster).
- Split the `index.rst` into multiple `.rst` files and use a `toctree` to have the sidebar populated in the sphinx_btd_theme.
- Replace `kevinpt/hdlparse` with `hdl/pyHDLParser` and `Hdlparse` with `pyHDLParser`; but not the references to PyPi.
- Break lines at max 120 chars.

The content of the README.rst is actually duplicated in the docs. We should probably remove most of the content from this file, and fix the syntax in `getting.rst` (see https://github.com/hdl/pyHDLParser/pull/2/files#diff-d6d6b4e691a98ef7d987710f8d3aeeed8eafecfbdfe0b990ece4d2a9bf81a347R16).